ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,83 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  mtc-apps-deployer:
+    name: mtc-apps-deployer
+    namespace: mtc-qe
+    tag: master
+  mtc-python-client:
+    name: mtc-python-client
+    namespace: mtc-qe
+    tag: master
+  openshift-cli-installer:
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+    mtc-apps-deployer:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-apps-deployer.tar.gz
+    mtc-python-client:
+      paths:
+      - destination_dir: .
+        source_path: /mtc-python-client.tar.gz
+  to: mtc-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtc-interop-aws-rosa
```

Comment:
I believe this PR is putting in place testing for Rosa Classic but I don't see anything that tells us that explicitly.

We use unique identifiers to help us separate our work

For self-managed we use "-lp-interop"
For Rosa HyperShift we us "-lp-rosa-hypershift"

For Rosa Classic what should we use?
- I'd reccomend "-lp-rosa-classic"

This unique identifier can either be in the filename or here in the test stanza name